### PR TITLE
First Assignment

### PR DESCRIPTION
**Pull request by Harsshath23**

### CODE REVIEW

File: Harsshath/Assignment1.scala [hunk 1]
```diff
@@ -0,0 +1,17 @@
+object Assignment1 extends App{
+  println("Size of the Array")
+  val n = scala.io.StdIn.readInt()
+  var arr = new Array[Int](n)
+  println("Elements of the Array")
+  var count = 0
```
Comment (tg-trainer) [inline Harsshath/Assignment1.scala:6]: Dont use var.

Comment (Harsshath23) [reply Harsshath/Assignment1.scala:6]: Ok I will update.

Comment (tg-trainer) [reply Harsshath/Assignment1.scala:6]: Thanks, try with recursion.

Comment (Harsshath23) [reply Harsshath/Assignment1.scala]: Can you check now I tried code using recursion

**Comment by Harsshath23** [comment]
Modified code using recursive function

**Pull request merged**